### PR TITLE
fix for member not redirecting to dir search once tos is accepted

### PR DESCRIPTION
**Pull request by nitheesh-aot**
*Description of changes:*
fix for member not redirecting to dir search once tos is accepted

*Checklist:*
I confirm that below checklist items are addressed in this pull request; (check the boxes applicable)
- [ ] No lint errors
- [ ] No test case failures and proper coverage for all modules/classes
- [ ] Updated the deployment configs for new environment variable(s)
- [ ] Updtaed the postman collection in entity repository (https://github.com/bcgov/entity/tree/master/api-e2e/postman) for e2e tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/mixins/NextPageMixin.vue [hunk 1]
```diff
@@ -49,7 +49,9 @@ export default class NextPageMixin extends Vue {
   protected getNextPageUrl (): string {
     switch (this.currentUser?.loginSource) {
       case LoginSource.IDIR:
-        return `/${Pages.SEARCH_BUSINESS}`
+        let idirNextStep = '/'
```
Comment (sutherlanda) [inline auth-web/src/components/auth/mixins/NextPageMixin.vue]: Don't this change is necessary? Maybe leftover from something?

File: auth-web/src/views/auth/TermsOfServiceView.vue [hunk 2]
```diff
@@ -89,6 +89,10 @@ export default class TermsOfServiceView extends Mixins(NextPageMixin) {
     this.atBottom = (e.target.scrollHeight - e.target.scrollTop) <= (e.target.offsetHeight + 25)
   }
 
+  mounted () {
+    this.$store.commit('updateHeader')
```
Comment (sutherlanda) [inline auth-web/src/views/auth/TermsOfServiceView.vue]: Shouldn't need to do this, is something not updating?

File: auth-web/src/components/auth/mixins/NextPageMixin.vue (hunk 1 above)
Comment (nitheesh-aot) [reply auth-web/src/components/auth/mixins/NextPageMixin.vue]: I will change that

File: auth-web/src/views/auth/TermsOfServiceView.vue (hunk 2 above)
Comment (nitheesh-aot) [reply auth-web/src/views/auth/TermsOfServiceView.vue]: Its needed, the account was not updating without it

Comment (sutherlanda) [reply auth-web/src/views/auth/TermsOfServiceView.vue]: Okay - I may try to find a better way to do this.  Forcing a header remount is expensive.

**Pull request merged**